### DISCUSSION TITLE
post/pluralsight-updates-2021

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#11] [Announcements]
# post/pluralsight-updates-2021

Mark Heath's Development Blog

https://markheath.net/post/pluralsight-updates-2021

**Comment by HolisticDeveloper** [comment]
I've just been watching your Pluralsight course on Durable Functions, which has been quite helpful! You mentioned on this post something about running Durable Functions in a containerized environment (rather than Azure). I've only recently become aware of this capability for Azure Functions. Are you aware of any good courses that cover this?

**Reply by markheath** [reply]
Glad you've found the course helpful. I don't know of any courses specifically about that I'm afraid. It's a relatively new feature and has only recently become a viable option outside Azure by the addition of different supported backing stores ("[storage providers](https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-storage-providers)") for Durable Functions (previously it needed to use Azure Storage, but now can use SQL or Netherite instead. And you might also want to look into KEDA which gives you the serverless scale out capabilities of Azure within a k8s cluster.
